app/src/main/java/com/github/se/eventradar/viewmodel/ReusableEnums.kt

```diff
@@ -0,0 +1,6 @@
+package com.github.se.eventradar.viewmodel
+
+enum class ViewType {
```

Comment:
Could this be accomplished with just one boolean variable?

Reply:
Yes it can, i think that would make more sense too. Will change.